src/utils/screenshot.util.ts

```diff
@@ -0,0 +1,56 @@
+import {PNG, PNGWithMetadata} from 'pngjs';
+import * as fs from 'fs';
+import Pixelmatch from 'pixelmatch';
+
+import {ElementFinder} from "../wdio";
+import {ReportUtility} from "./report.util";
+import {AssertionUtility} from "./assertion.util";
+import {Widget} from "../widgets";
+import {BasePage} from "../pages";
+
+
+export class ScreenshotUtility {
+    private static basePath = './screenshots';
+    private static pixelTolerance = 0;
+
+    public static setBasePath(basePath: string): void {
+        this.basePath = basePath;
+    }
+
+    public static setPixelThreshold(pixelTolerance: number): void {
```

Comment:
This method should be better named as `setPixelTolerance`, so it better matches with the associated member variable

Reply:
That makes sense, added in next commit